best-first-cv/metadata.json

```diff
@@ -38,13 +38,30 @@
       "type": "list",
       "default": [],
       "description": "List of field IDs to be pre-selected"
+    },
+    {
+      "name": "early-stop-performance",
+      "type": "number",
+      "default": 1,
+      "description": "Performance improvement value (in %) used as a threshold to consider good performance of a feature"
+    },
+    {
+      "name": "max-bad-percentage",
+      "type": "number",
+      "default": 20,
+      "description": "(%) Maximum number of iterations with bad performance, as percentage of the number of initial features"
```

Comment:
nit: you already say it's a percentage, so the `(%)` prefix is kind of redundant... if you still like it, then it should be used too in the previous input's description, for consistency.